src/modules/auth/auth.controller.ts

```diff
@@ -58,4 +60,24 @@ export default class RegistrationController {
     const { id: user_id } = request['user'];
     return this.authService.enable2FA(user_id, password);
   }
+
+  @skipAuth()
```

Comment:
If this route is protected, remove this decorator

Reply:
This route is not protected